likeminds-chat-reactnative-ui/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "likeminds_chat_reactnative_ui",
+  "versionCode": "23",
+  "versionName": "1.8.0",
```

Comment:
I need to check about these version code and version name

Reply:
changed `versionCode` to `1` and `versionName` to `0.5.0`